infrastructure/src/main/java/org/corfudb/infrastructure/ServerContext.java

```diff
@@ -662,6 +664,22 @@ private EventLoopGroup getNewBossGroup() {
         }
     }
 
+    /**
+     * Get a new instance of {@link StreamLogParams} representing the stream log parameters.
+     *
+     * @return an instance of {@link StreamLogParams}
+     */
+    public StreamLogParams getStreamLogParams() {
+        return StreamLogParams.builder()
+                .logPath(getServerConfig(String.class, "--log-path"))
+                .verifyChecksum(!getServerConfig(Boolean.class, "--no-verify"))
+                .build();
+    }
+
+    public StreamLogDataStore getStreamLogDataStore() {
```

Comment:
Is singleton better here?

Reply:
Right now I don't see a strong reason to make it singleton.